libs/coin-modules/coin-aptos/src/bridge/logic.ts

```diff
@@ -169,6 +171,88 @@ export const txsToOps = (
   return [ops, opsTokens];
 };
 
+export function txsToApiOps(
```

Comment:
Shouldn't it be on src/logic path? It will be used by Alpaca API and the Bridge, right?